### DISCUSSION TITLE
Lesson 5: how can I create a new directory in python 3?

### DISCUSSION THREAD

**Discussion by HalaAura** [smartcontractkit/full-blockchain-solidity-course-py#845] [Q&A] [answered]
Hello! 
I just didn't know how to create a new directory. Using (cd ..) didn't work. That's at the beginning of the lesson 5.
Would you please help me? 

Thank you so much! :)

**Comment by Verde888** [comment, marked as answer]
Hi @HalaAura!
I'm not sure if this is any help but:
The command "cd .." moves you one level up in the folder structure (he left Web3_py_simple_storage this way).
"mkdir" creates a new folder in the folder where you are currently. 
Just in case, use the "dir"  command to see what's in the current folder. 
Use "cd" and a folder name to enter the chosen folder.

So you should be in /demos or any other folder you want to create your brownie project and then use the "mkdir brownie_simple_storage" command. 

In fact he does it in the VSCode console but it is not necessary, you can create such a new folder the way you would do it normally on your computer and then just open it with VSCode.

edit: It's is not really about creating a new directory "in python 3", those are standard commands.  They will not work if you are for example in the Python shell. You can try to reinitiate the terminal (killing it with the 'bin' button).
